common/src/main/java/net/bettercombat/config/ServerConfig.java

```diff
@@ -52,4 +56,11 @@ public class ServerConfig implements ConfigData {
     public float getUpswingMultiplier() {
         return Math.max(0.2F, Math.min(1, upswing_multiplier));
     }
+
+    public boolean isEntityHostileVehicle(String entityName){
```

Comment:
I like the solution overall, but the name of this function gave me an idea.
What if hostile vehicles would be checked not against ID, but rather against `instanceof HostileEntity`.

What do you think? Would that work?

Reply:
This makes sense to me in principle, but my concern there is that other modders may not implement things in a way that we can rely on. 

I like the 'fallback compatibility' in this mod, so I wonder if this could follow a similar pattern and short circuit from instanceof HostileEntity and then check a blacklist?

Reply:
I pushed a commit that short circuits from one to the other. We could potentially do things that way.